include/mbedtls/mbedtls_config.h

```diff
@@ -1129,7 +1129,7 @@
  *
  * Enable support for PKCS#1 v1.5 encoding.
  *
- * Requires: MBEDTLS_RSA_C
+ * Requires: MBEDTLS_MD_C, MBEDTLS_RSA_C
```

Comment:
Side note, pre-existing: `library/rsa.c` is really confusing as it has 
```c
#if defined(MBEDTLS_PKCS1_V21)
#include "mbedtls/md.h"
#endif
```
while it already include `rsa.h` which already includes `md.h` unconditionally, so this conditional include gives the wrong impression. Would you mind fixing it while at it?

Other note: unless I missed it, `md.c` is only used to get the size for a given alg for PKCS#1 v1.5, so that dependency could easily be removed by hardcoding the sizes in a switch statement instead it desired. (Not asking to do it, just noting it could be done.)

Reply:
Indeed `rsa.c` includes `rsa.h` that includes `md.h`. So mentioned code snippet can be removed.